packages/appsync-emulator-serverless/dynamodbSource.js

```diff
@@ -7,12 +7,23 @@ const nullIfEmpty = obj => (Object.keys(obj).length === 0 ? null : obj);
 const unmarshall = (raw, isRaw = true) => {
   const content = isRaw ? Converter.unmarshall(raw) : raw;
 
-  // because of the funky set type used in the aws-sdk we need to further unwrap
+  // Because of the funky set type used in the aws-sdk, we need to further unwrap
   // to find if there is a set that needs to be unpacked into an array.
+
+  // Unwrap sets
   if (content && typeof content === 'object' && content.wrapperName === 'Set') {
     return content.values;
   }
 
+  // Unwrap lists
```

Comment:
So here's the actual fix. I realize the lines below it do some work with objects as well. Perhaps they could be updated to resolve the issue. But adding separate logic was the most straightforward.